resolving/resolver-base/src/index.ts

```diff
@@ -38,21 +38,69 @@ export type Resolution =
   GitResolution |
   ({ type: string } & object)
 
-export interface ResolveResult {
+export type ResolveVia =
```

Comment:
to be consistent with the field name
```suggestion
export type ResolvedVia =
```